packages/website/pages/terms.js

```diff
@@ -23,75 +23,158 @@ export default function TermsOfService() {
           be stored on IPFS, this website incorporates the{' '}
           <a className="black" href="https://discuss.ipfs.io/tos">
             Terms of Service of IPFS.io
           </a>
           .
         </p>
-        <p className="lh-copy">
-          If you do not agree to all the terms and conditions of this agreement,
-          then you may not access the Website or use any services.
-        </p>
+        
         <h2 className="chicagoflf">
           <HashLink id="storage-term">Storage Term</HashLink>
         </h2>
         <p className="lh-copy">
-          Data will be stored at no cost to the user on IPFS for as long as
-          Protocol Labs, Inc. continues to offer free storage for NFT’s. Data
-          will continue to be persisted ad infinitum <strong>or</strong> until
-          Protocol Labs decides to conclude the NFT.storage project. Prior to
-          termination, Protocol Labs will provide 90 days notice to users via
-          email to allow users enough time to make arrangements for storing
-          their data by other means.
+          All data is stored redundantly on both the IPFS and Filecoin networks, 
+          and guaranteed to be publicly available and verifiable via its unique 
+          content identifier.
+        </p>
+        <p className="lh-copy">
+          All data stored on {' '}
+          <a className="black" href="https://nft.storage">
+            NFT.Storage
+          </a>{' '} is stored in long-term (18 month) deals 
+          with independent Filecoin Storage Providers. Filecoin storage deals can 
+          be renewed by anyone, are non-custodial, and anyone can access information 
+          about them (content address, duration and expiration, price, etc.) and 
+          verify the data is legitimately stored via the Filecoin blockchain. 
+          All Filecoin deals are{' '}
+          <a className="black" href="https://github.com/filecoin-project/filecoin-plus-client-onboarding">
+            Verified Deals
+          </a>{' '}
+          , which Filecoin Storage Providers are incentivized to accept at a lower 
+          price vs. other deals because they have a relatively higher increase in 
+          their chance to win block rewards.
+        </p>
+        <p className="lh-copy">
+          {' '}
+          <a className="black" href="https://nft.storage">
+            NFT.Storage
+          </a>{' '} is committed to ensuring that all NFTs on Filecoin have a 
+          perpetual storage solution. All Filecoin storage deals facilitated by 
+          {' '}
+          <a className="black" href="https://nft.storage">
+            NFT.Storage
+          </a>{' '} are automatically renewed in perpetuity by {' '}
+          <a className="black" href="https://nft.storage">
+            NFT.Storage
+          </a>{' '} before 
+          expiration. In late 2022, we plan to build tooling to enable the 
```

Comment:
Sounds a bit too too much like a "rough roadmap".

How about something like:

> NFT.Storage will switch to non-custodial, perpetual renewal of storage contracts independent of NFT.Storage to secure the perpetual persistence of all NFTs without dependence on any single entity when the tooling becomes available in the Filecoin network.

Reply:
yeah agreed - it was a direct ask from Pooja to include a date, but will make it more subtle